platforms/extensibility/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/impldeps/GradleImplDepsShadingIssuesIntegrationTest.groovy

```diff
@@ -38,7 +38,9 @@ class GradleImplDepsShadingIssuesIntegrationTest extends BaseGradleImplDepsInteg
 
                 void apply(Project project) {
                     def conf = project.configurations.create('bug')
-                    project.${mavenCentralRepository()}
+                    project.repositories {
```

Comment:
Ah, it's because I've wrapped url with`uri()` in `RepoScriptBlockUtil`. In should be `project.uri` since here it's not a build script. I've reverted the change here and in `RepoScriptBlockUtil`